### PR TITLE
Add configuration package reference because Logging dropped it

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: src/EFCore.Relational/EFCore.Relational.csproj [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="$(CoreFxVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="$(AspNetCoreVersion)" />
```
Comment (natemcmaster) [inline src/EFCore.Relational/EFCore.Relational.csproj]: Is config.abstractions sufficient?

Comment (pakrym) [inline src/EFCore.Relational/EFCore.Relational.csproj]: Yes, but need to fix some test projects. Working on it

**Comment by pakrym** [comment]
:up::date:

**Comment by pakrym** [comment]
Merging to unblock the build

**Pull request merged**